src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2642 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class AccountCreateParams extends ApiRequestParams {
+  /**
+   * An [account token](https://stripe.com/docs/api#create_account_token), used to securely provide
+   * details to the account.
+   */
+  @SerializedName("account_token")
+  String accountToken;
+
+  /**
+   * Either a token, like the ones returned by [Stripe.js](https://stripe.com/docs/stripe.js), or a
+   * dictionary containing a user's bank account details.
+   */
+  @SerializedName("bank_account")
+  Object bankAccount;
```

Comment:
Why is this one an Object when `Company` is a class. I'm assuming it's because it has different shapes but what does the code using it looks like?

Also, I'm realizing that `bank_account` should not be supported at all. Only `external_account` should be supported. `bank_account` was removed in 2015 in theory: https://stripe.com/docs/upgrades#2015-10-01

Reply:
This is removed and now we have `externalAccount` String token instead.
The documentation is still wrong though. I have a PR for gated docs that doesn't change that in API refs (saying that it's still a hash)